packages/e2e-tests/specs/editor/various/embedding.test.js

```diff
@@ -307,7 +307,7 @@ describe( 'Embedding content', () => {
 		await page.keyboard.type( 'Hello there!' );
 		await publishPost();
 		const postUrl = await page.$eval(
-			'[id^=inspector-text-control-]',
+			'.editor-post-publish-panel [id^=inspector-text-control-]',
```

Comment:
That's the interesting thing! For the longest time, this test was failing for this PR.

The reason for that is because the `Paragraph` block adds a `TextControl` component.
When this test `$eval` for an input, it finds the new LineHeightControl, **not** the publish URL input.

To resolve this, I made the test more specific to the Publish Panel

Reply:
Thank you for clarification @ItsJonQ!